src/MiNET/TestPlugin/CoreCommands.cs

```diff
@@ -530,10 +531,12 @@ private void RewriteBiome(Player player)
 			//var biomeIds = new byte[] {8, 170, 171, 172, 173};
 			var biomeIds = new byte[] {8, 170, 171, 172, 173};
 			byte biomeId = biomeIds[new Random().Next(biomeIds.Length)];
-			for (int i = 0; i < chunk.biomeId.Length; i++)
-			{
-				chunk.biomeId[i] = biomeId;
-			}
+
+			// TODO - 1.20 - update
+			//for (int i = 0; i < chunk.biomeId.Length; i++)
+			//{
+			//	chunk.biomeId[i] = biomeId;
```

Comment:
Any chance you could keep this working?

Reply:
Why not? But before that, I need to rework data storage in chunks, since the current one is outdated and inconvenient.

When I'm done, it will turn into something like:

chunk.Biomes.Fill(id);

Reply:
should it be active?